docs/source/configuring.mdx

```diff
@@ -8,10 +8,11 @@ description: Configure Rover CLI for interaction with GraphOS. Learn to authenti
 
 ### 1. Obtain an API key
 
-All Rover commands that communicate with [GraphOS](/graphos/) require an API key to do so. GraphOS supports two types of API keys: _personal API keys_ and _graph API keys_.
+All Rover commands that communicate with [GraphOS](/graphos/) require an API key to do so. GraphOS supports three types of API keys: _graph API keys_, _subgraph API keys_, and _personal API keys_.
+
```

Comment:
```suggestion
All Rover commands that communicate with [GraphOS](/graphos/) require an API key. GraphOS supports the following types of API keys: graph API keys, subgraph API keys, and personal API keys.
```
Librarian suggestion plus removing number of keys reference to future proof. Remove extra new line.